src/server/game/Handlers/Socialhandler.cpp

```diff
@@ -113,7 +113,7 @@ void WorldSession::HandleAddIgnoreOpcode(WorldPacket& recv_data)
         return;
 
     uint64 IgnoreGuid = MAKE_NEW_GUID(lowGuid, 0, HIGHGUID_PLAYER);
-    FriendsResult ignoreResult = FRIEND_IGNORE_NOT_FOUND;
+    FriendsResult ignoreResult;
```

Comment:
this value was never used, it will be either `FRIEND_IGNORE_SELF`, `FRIEND_IGNORE_ALREADY` or `FRIEND_IGNORE_FULL` (see below)

Reply:
Yes, but I decided to leave everything as is.